jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -690,9 +704,13 @@ class DeploymentFlow extends React.Component {
   */
   _generateExpertBudgetSection() {
     const status = this._getSectionStatus('expert-budget');
+    const state = this.state;
     if (!status.visible) {
       return;
     }
+    const estimate = parseInt(state.ddEntity.toEntity().price, 10);
+    const hourPrice = (state.selectedSLA && state.selectedSLA.hourPrice) || 0;
+    const machineCount = state.ddEntity.get('machineCount') || '1';
```

Comment:
You're doing maths with this count further down so should the `1` not be a string?

Reply:
yep, done